### ISSUE TITLE
Option to only track devtools console time

### ISSUE THREAD

**Issue by ohitstom** [wakatime/chrome-wakatime#231]
new feature request:

ispend alot of my time troubleshooting javascript code from chrome devtools, preferably this would be implemented using https://developers.google.com/cast/docs/debugging/remote_debugger so that other apps can be recognized!

**Comment by alanhamlett** [comment]
So only track time in browser when devtools is open, otherwise disable WakaTime tracking in the browser?

**Comment by ohitstom** [comment]
> So only track time in browser when devtools is open, otherwise disable WakaTime tracking in the browser?

That would be a simpler implementation yes, though i meant more that it could interfact with remote debugger to get status on when you are typing in devtools (if possible) and for how long for higher accuracy and compatibility with desktop apps.

Though im not sure if that would even be in-line with the browser version?

**Comment by alanhamlett** [comment]
> get status on when you are typing in devtools

The browser extension should already do that. When devtools is open it tracks time towards `Debugging` category instead of `Browsing`.

**Comment by ohitstom** [comment]
> > get status on when you are typing in devtools
> 
> The browser extension should already be doing that. When devtools is open it tracks time towards `Debugging` category instead of `Browsing`.

true, but it would be nice to be able to have debugging tracked when debugging applications like discord or Spotify which have remote debugging ports!

a browser implementation would still be great though
